docs/content/CONTRIBUTING.adoc

```diff
@@ -123,14 +123,66 @@ are related to "getting started", easier maintenance, ....
 ____
 
 
-=== Alias when renaming pages
+=== Use alias to create redirects
 
-IMPORTANT: Impact on SEO
+**IMPORTANT**: not creating redirects impacts the SEO and the reader experience
 
-See
+You **MUST** create an alias when you do the following actions on a existing page
+
+* rename
+* move
+* delete
+
+When doing such actions, the former url of the page is no more available, so its content is no more available for readers (http 404).
+The solution is to setup a redirect that will lead the reader to a new destination.
+
+Antora provides the 'Alias' feature to deal with redirects
+
+* documentation writer knows where to redirect, so it create an alias in the target page that references the old page
+* the documentation build process creates a redirect for each alias
+* the reader is happy and always find the information in the documentation
+
+Limitation of aliases (https://gitlab.com/antora/antora/-/issues/806)
+> Page aliases really were designed to address page renames/moves, particularly within a single component version. They aren't a general-purpose URL router. They can be used for more than what I described, but then you really do have to think about what you are doing and use with care...because it can quickly become confusing.
```

Comment:
⚠️ markdown syntax, should use an AsciiDoc block or https://docs.asciidoctor.org/asciidoc/latest/blocks/blockquotes/#basic-quote-syntax

Reply:
fixed in 94182b5